nsfailover.sh

```diff
@@ -148,6 +148,7 @@ search ${NS_SEARCH}"
 
 # Load current config (without comments)
 current="$(cat "${NS_FILE}" |egrep -v '^#')" || true
+resolvconf=$(echo "$resolvconf" | egrep -v '^#')
```

Comment:
This seems to remove only `# Do not edit. Updated by cron process!!"` which is useful to have in that file tho?

Reply:
You are correct. This line only remove the comments, but without this the if was returning true ever time and execute copy of /etc/resolv.cont to .bak causing a possible DoS on host. My patch correct this.

Reply:
A variable resolvconf and content of resolv.conf every are diferent If you not remove comments in both.

Reply:
what about adding a `commentLessResolvConf` var or similar, that we run the compare against, but that we do not write to disk?

Reply:
The file write disk have all coment necessary,  but only have change on the file. Insert this var is a option.

Reply:
Hi, I review my code here and adding a commentLessResolveConf for test with $current, resolve the problem.
I make this and are ok now.